akka-docs/src/main/paradox/typed/distributed-data.md

```diff
@@ -569,8 +569,8 @@ Implement the additional methods of @scala[`DeltaReplicatedData`]@java[`Abstract
 #### Serialization
 
 The data types must be serializable with an @ref:[Akka Serializer](../serialization.md).
-It is highly recommended that you implement  efficient serialization with Protobuf or similar
-for your custom data types. The built in data types are marked with `ReplicatedDataSerialization`
+It is highly recommended that you implement efficient serialization with Protobuf or similar
+for your custom data types, otherwise inefficient Java serialization applies. The built in data types are marked with `ReplicatedDataSerialization`
```

Comment:
By default java serialization is disabled so I don't see how that can be the case unless a user opted in to it?

Reply:
@johanandren  I see. I am still at 2.5.x where java serialization is enabled by default, that's why.

Then if the user didn't implement its own, then by default the code will simply not work in 2.6.x, is that correct? do you think we can make it explicit in the doc?

Reply:
Yes, if the user didn't pull in jackson and set that up, they would have to set some other serialization up. Perhaps it could be something like: `You will need to set up @ref[serialization](wherever/serialization.md is) for your custom data types. It is highly recommended that you choose an efficient serialization implementation.`